core/executor/src/system.rs

```diff
@@ -44,6 +44,7 @@ fn revert_resp(gas_limit: U256) -> TxResp {
         ret:          vec![],
         gas_used:     1u64,
         remain_gas:   (gas_limit - 1).as_u64(),
+        fee_cost:     U256::one(),
```

Comment:
why not `gas_used * gas_price`

Reply:
The transactions that call system script only pay one gas and the gas price is one. Because of these transactions can only be sent by validator nodes.